core/src/main/java/io/grpc/internal/AutoConfiguredLoadBalancerFactory.java

```diff
@@ -164,28 +157,29 @@ void setDelegate(LoadBalancer lb) {
     }
 
     @VisibleForTesting
-    LoadBalancer.Factory getDelegateFactory() {
-      return delegateFactory;
+    LoadBalancerProvider getDelegateProvider() {
+      return delegateProvider;
     }
 
     /**
      * Picks a load balancer based on given criteria.  In order of preference:
      *
      * <ol>
      *   <li>User provided lb on the channel.  This is a degenerate case and not handled here.</li>
-     *   <li>gRPCLB if on the class path and any gRPC LB balancer addresses are present</li>
-     *   <li>RoundRobin if on the class path and picked by the service config</li>
-     *   <li>PickFirst if the service config choice does not specify</li>
+     *   <li>"grpclb" if on the class path and any gRPC LB balancer addresses are present</li>
```

Comment:
s/on the class path/in the registry/

Although actually, it appears that's not part of the conditions below. So maybe:

> "grpclb" if any gRPC LB balancer addresses are present

Reply:
Done.